Hi @lbailly, thanks for PR! Can you please check code style suggestions
```
0.04s$ golint -set_exit_status ./...
payment.go:4:2: exported const RECURRING should have comment (or a comment on this block) or be unexported
payment.go:40:2: struct field IssuerUrl should be IssuerURL
payment.go:56:6: exported type Recurring should have comment or be unexported
payment3d.go:3:1: comment on exported type Authorise3D should be of the form "Authorise3D ..." (with optional leading article)
payment3d_gateway.go:11:1: comment on exported method PaymentGateway.Authorise3D should be of the form "Authorise3D ..."
Found 5 lint suggestions; failing.
```

and update PR. Thanks in advance!

--- a/payment.go
+++ b/payment.go
@@ -1,9 +1,9 @@
 package adyen
 
+/* One-click functionality gives the shopper the option to store their payment details with the merchant, within the Adyen environment. In this type of transaction, the shopper needs to enter the CVC code for the transaction to get through.
+Same than for ONECLICK except than the shopper doesn't need to enter the CVC */
 const (
-	// One-click functionality gives the shopper the option to store their payment details with the merchant, within the Adyen environment. In this type of transaction, the shopper needs to enter the CVC code for the transaction to get through.
 	ONECLICK  = "ONECLICK"
-	// Same than for ONECLICK except than the shopper doesn't need to enter the CVC
 	RECURRING = "RECURRING"
 )
 
@@ -14,21 +14,21 @@ type AuthoriseEncrypted struct {
 	Reference       string          `json:"reference"`
 	MerchantAccount string          `json:"merchantAccount"`
 	// Mandatory for recurring payment
-	ShopperReference string `json:"shopperReference,omitempty"`
-	Recurring       *Recurring      `json:"recurring,omitempty"`
+	ShopperReference string     `json:"shopperReference,omitempty"`
+	Recurring        *Recurring `json:"recurring,omitempty"`
 	// Required for a 3DS process
 	BrowserInfo *BrowserInfo `json:"browserInfo,omitempty"`
 }
 
 // Authorise structure for Authorisation request (card is not encrypted)
 type Authorise struct {
-	Card            *Card      `json:"card"`
-	Amount          *Amount    `json:"amount"`
-	Reference       string     `json:"reference"`
-	MerchantAccount string     `json:"merchantAccount"`
+	Card            *Card   `json:"card"`
+	Amount          *Amount `json:"amount"`
+	Reference       string  `json:"reference"`
+	MerchantAccount string  `json:"merchantAccount"`
 	// Mandatory for recurring payment
-	ShopperReference string `json:"shopperReference,omitempty"`
-	Recurring       *Recurring `json:"recurring,omitempty"`
+	ShopperReference string     `json:"shopperReference,omitempty"`
+	Recurring        *Recurring `json:"recurring,omitempty"`
 	// Required for a 3DS process
 	BrowserInfo *BrowserInfo `json:"browserInfo,omitempty"`
 }

--- a/payment.go
+++ b/payment.go
@@ -1,8 +1,10 @@
 package adyen
 
 const (
-	RECURRING = "RECURRING"
+	// One-click functionality gives the shopper the option to store their payment details with the merchant, within the Adyen environment. In this type of transaction, the shopper needs to enter the CVC code for the transaction to get through.
 	ONECLICK  = "ONECLICK"
+	// Same than for ONECLICK except than the shopper doesn't need to enter the CVC
+	RECURRING = "RECURRING"
 )
 
 // AuthoriseEncrypted structure for Authorisation request (with encrypted card information)
@@ -37,9 +39,9 @@ type AuthoriseResponse struct {
 	ResultCode    string `json:"resultCode"`
 	AuthCode      string `json:"authCode"`
 	RefusalReason string `json:"refusalReason"`
-	IssuerUrl string `json:"issuerUrl"`
-	MD string `json:"md"`
-	PaRequest string `json:"paRequest"`
+	IssuerURL     string `json:"issuerUrl"`
+	MD            string `json:"md"`
+	PaRequest     string `json:"paRequest"`
 }
 
 // AdditionalData stores encrypted information about customer's credit card
@@ -53,6 +55,7 @@ type BrowserInfo struct {
 	UserAgent    string `json:"userAgent"`
 }
 
+// Recurring hold the behavior for a future payment : could be ONECLICK or RECURRING
 type Recurring struct {
 	Contract string `json:"contract"`
 }

--- a/payment3d.go
+++ b/payment3d.go
@@ -2,9 +2,9 @@ package adyen
 
 // Authorise3D structure for Authorisation request (card is not encrypted)
 type Authorise3D struct {
-	MD string `json:"md"`
-	MerchantAccount string     `json:"merchantAccount"`
-	BrowserInfo *BrowserInfo `json:"browserInfo"`
-	PaResponse string `json:"paResponse"`
-	ShopperIP string `json:"shopperIP,omitempty"`
+	MD              string       `json:"md"`
+	MerchantAccount string       `json:"merchantAccount"`
+	BrowserInfo     *BrowserInfo `json:"browserInfo"`
+	PaResponse      string       `json:"paResponse"`
+	ShopperIP       string       `json:"shopperIP,omitempty"`
 }

--- a/payment3d.go
+++ b/payment3d.go
@@ -1,6 +1,6 @@
 package adyen
 
-// Authorise structure for Authorisation request (card is not encrypted)
+// Authorise3D structure for Authorisation request (card is not encrypted)
 type Authorise3D struct {
 	MD string `json:"md"`
 	MerchantAccount string     `json:"merchantAccount"`

--- a/payment3d_gateway.go
+++ b/payment3d_gateway.go
@@ -8,7 +8,7 @@ type Payment3DGateway struct {
 // authorise3DType - authorise type request, @TODO: move to enums
 const authorise3DType = "authorise3d"
 
-// Authorise - Perform authorise payment in Adyen
+// Authorise3D - Perform authorise payment in Adyen
 func (a *PaymentGateway) Authorise3D(req *Authorise3D) (*AuthoriseResponse, error) {
 	resp, err := a.execute(authorise3DType, req)
 

